errors/react-hydration-error.md

```diff
@@ -36,6 +36,30 @@ function MyComponent() {
   return <h1 className={`title ${color}`}>Hello World!</h1>
 }
 ```
+Another example:
+NextJS is not comfortable having a p tag wrapping your divs, sections etc so it will yell "Hydration failed because the initial UI does not match what was rendered on the server". 
```

Comment:
The wording here should be changed.

It is **React that does not** tolerate having a `div` inside `p`. **Because having a `div` inside `p` has violated the HTML5 spec**. Although the browser might tolerate it, React will throw an error to maintain a consistent result across browsers and avoid undefined behavior.

The hydration error is completely not related to Next.js, it is all caused by the developer not following the HTML5 spec.